tests_e2e/test_selenium.py

```diff
@@ -70,15 +42,16 @@ def driver():
         yield driver
 
 
-def test_task_journey(driver, app_with_temp_board):
+def test_task_journey(driver, app_with_temp_database):
```

Comment:
Given how many places you've changed this, it could be one to look at [switching the fixture to autouse](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request) so that we can clear these references out and we don't have to update them all as often in future